source/_integrations/waze_travel_time.markdown

```diff
@@ -62,6 +62,62 @@ template:
 ```
 
 {% endraw %}
+Automations / Scrips can be setup to poll Waze on any changes to the input_select.
+An example of a script:
+```
```

Comment:
This here is why the bot is complaining. You have started a code block but haven't specified the language. It should be ```yaml